src/test/java/com/easypost/BillingTest.java

```diff
@@ -22,17 +36,38 @@ public static void setup() throws EasyPostException {
         vcr = new TestUtils.VCR("billing", TestUtils.ApiKey.PRODUCTION);
     }
 
+    /**
+     * Release the static mock once it has been used.
+     */
+    @AfterAll
+    public static void cleanup() {
+        requestMock.close();
+    }
```

Comment:
This function should be at the top of this file so it's easy to find and so it's not mixed in with the rest of the tests.

Reply:
This function is on the top of this file, it should come after `setup()` which has `BeforeAll` annotation. 
`AfterAll` function should come after the `BeforeAll` function.

Reply:
Ah, showed up weird on GitHub. Thanks!